crypto/bio/bio_print.c

```diff
@@ -293,6 +298,17 @@ _dopr(char **sbuffer,
                            flags, G_FORMAT))
                     return 0;
                 break;
+#else
+            case 'f':
+            case 'E':
+            case 'e':
+            case 'G':
+            case 'g':
+                /* not implemented for UEFI */
+                ERR_raise(ERR_LIB_BIO, ERR_R_UNSUPPORTED);
+                return 0;
+                break;
```

Comment:
Remove this line please.  It's unreachable and compilers/static analysers will complain.